assets/css/app.css

```diff
@@ -21,4 +21,16 @@
 .file-input-container {
   display: inline-block;
   position: relative;
+}
+
+.b {
+  white-space: nowrap; 
+  width: 50px; 
+  overflow: hidden;
+  text-overflow: ellipsis; 
+}
+
+.assignment-test-sidebar {
+  width: 180px;
```

Comment:
We don't have to create these kind of classes because tailwind already has such functions.